src/components/DataSetControls.vue

```diff
@@ -204,7 +205,7 @@ export default {
 }
 </script>
 
-<style scoped>
+<style>
```

Comment:
why not scoped?

Reply:
should be scoped, was a test. Setting this back